src/ui/components/Button/Button.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { theme } from '../../../shared/theme';
+import ButtonUrl from '../../../public/img/silverButton.png';
+
+const BUTTON_HEIGHT_SIZE = '8rem';
+const BUTTON_WIDTH_SIZE = '21.25rem';
+const BUTTON_FONT_SIZE = '22px';
```

Comment:
Wydaje mi się, że rozmiar fontu też powinien być w remach

Reply:
Font chyba lepiej zrobić w 'vw' aby był responsywny. To zmienie faktycznie

Reply:
To wtedy nie będzie dziwnie wyglądało? Szerokość i wysokość buttona w remach, więc rozmiar buttona nie będzie się zmieniał wraz z szerokością viewportu, a rozmiar fontu tak

Reply:
może tak być w sumie. To może na razie zostawmy te px, a później się zrobi refaktor wszystkich fontów, bo i tak te co są w src/public/index.css są w 'px'

Reply:
Właśnie te, które są w indeksie to "definicja" rozmiaru `1rem`, więc on powinny być w pikselach (nie możemy definiować `rem`a za pomocą `rem`a tak na logikę).

A w tym miejscu raczej @mmejer ma rację, bo teraz (chyba, nie testowałem) powyżej 1920 szerokości ekranu przycisk się powiększy a jego tekst nie, ale to też nie jakiś koniec świata.